School.js

```diff
@@ -21,6 +26,7 @@ class School {
     this.classes[name] = newClass;
   }
 
+
```

Comment:
`this.classes[name] = newClass` is not correct. You can either populate your `this.classes` object with information, or leave it empty and include this line in your addClass function.